### PR TITLE
Setup capstone project

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
The things that I did in this PR:

1. No linters error
2. Using Bootstrap and CSS for styling and layout
3. Using vanilla js for dynamic parts of the project
4. Do Githubflow and branching
5. Write proper commit messages
6. Make a Loom video for other people who are new to this project
7. Write proper documentation and workflow

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -49,19 +49,18 @@
...
+  background-color: #eee;
 }
 
-p {
+/* p {
   margin-top: 0;
-  margin-bottom: 0rem !important;
-}
+  margin-bottom: 0 !important;
+} */
```
Comment (GedeonTS) [inline assets/styles/style.css:64]: - Kindly make sure to remove commented out code to make your code clean and more professional :recycle:

File: index.html [hunk 2]
```diff
@@ -103,31 +188,33 @@
...
+              sharing, creating a positive change with people from over 110
+              countries is taking place in October,in Japan.
+            </p>
+          </div>
+          <!-- Headline date -->
+          <div class="headline-date-container px-3 fw-bolder fs-1 color1">
+            2024.10.15(THU) ~ 16(FRI)
+          </div>
+          <!-- Headline address -->
+          <div class="headline-address-container px-3 pb-5 color1">
```
Comment (GedeonTS) [inline index.html:214]: - Kindly make sure to put a background that is white as we can see it from Cindy's design below :point_down: , for this you can add another container with a white and a bit transparent background:
  - Your design:

Screenshot from 2023-04-22 00-01-16

  - Cindy's design:
Screenshot from 2023-04-21 23-52-13

File: index.html [hunk 3]
```diff
@@ -253,65 +346,66 @@ <h2 class="main-program-title text-black text-center mt-4 fw-bold">
...
-          src="https://img.icons8.com/external-royyan-wijaya-detailed-outline-royyan-wijaya/35/ec5242/external-chevron-arrow-line-royyan-wijaya-detailed-outline-royyan-wijaya.png" />
+          src="https://image.spreadshirtmedia.net/image-server/v1/compositions/T818A1PA1659PT17X26Y20D187054633W11924H12005/views/1,width=550,height=550,appearanceId=1,backgroundColor=FFFFFF,noPt=true/judo-kanji-emblem-in-kodokan-sign-baby-longsleeve-shirt.jpg"
+          alt=""
+          width="50px" />
+      </a>
+      <!-- Copyright -->
+      <div class="copyright-container">
+        <span class="copyright-date"></span> International Judo Federation. All
+        Rights Reserved
       </div>
```
Comment (GedeonTS) [inline index.html:401]: - Kindly make sure your footer's height is close to Cindy's, as we can see below, yours is still too short :point_down::
  - Your design:
Screenshot from 2023-04-22 00-10-01

  - Cindy's design:
Screenshot from 2023-04-22 00-07-32

File: about.html [hunk 4]
```diff
@@ -20,10 +20,333 @@
...
+        <div
+          class="logo-container border d-flex justify-content-center align-items-center w-75 mx-auto my-4 py-3">
+          <img
+            src="https://image.spreadshirtmedia.net/image-server/v1/compositions/T818A1PA1659PT17X26Y20D187054633W11924H12005/views/1,width=550,height=550,appearanceId=1,backgroundColor=FFFFFF,noPt=true/judo-kanji-emblem-in-kodokan-sign-baby-longsleeve-shirt.jpg"
+            alt=""
+            width="200px" />
+        </div>
+      </div>
+    </section>
+    <hr />
```
Comment (GedeonTS) [inline about.html:256]: - As suggested make the background white and the image is seen through it, as I suggested earlier, you can add another container and give it a white and a bit transparent background color:
Screenshot from 2023-04-22 00-10-01

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @M-AminAlizadeh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 

## Required Changes ♻️
- Good job recording a presentation video :clap: , however, you forgot to show the different pages on your website in live during the demo, kindly insert it :recycle: 
- I also suggest puting making when you deploy all your images displays to make it complete :recycle: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @M-AminAlizadeh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**